src/ex1/script_ex1.js

```diff
@@ -0,0 +1,141 @@
+const addButton = document.querySelector("#add-task-button");
+const triangle = document.querySelector("#triangle");
+const box = document.querySelector("#box");
+const welcome = document.querySelector("#welcome-sentence");
+const addTaskButton = document.querySelector(".task-button");
+const addTaskInput = document.querySelector(".task-input");
+const clearAllButton = document.querySelector("#clear-all-button");
+const sortByNameButton = document.querySelector("#sort-by-name-button");
```

Comment:
having all the elements outside here requires all the dom elements to exist when this file is loaded, which might not be the case in complex web apps that dynamically change the dom depending on the view